tailwind.config.js

```diff
@@ -10,24 +10,14 @@ export default {
     colors: {
       'black': '#1c1c1c',
       'borders': "#DDDDDD",
-      'bg':'#F3F5F7',
-      'danger-aler-bg': '#FC555619',
-      'unAssigned-bg': "#F7C2D2",
-      'Assigned-bg': "#99BFFB",
-      'InProgress-bg': "#FFE28C",
-      'Completed-bg': "#A1EDB1",
-      'placeholder':'#4D4D4D',
+      'gray-10':'#F3F5F7',
+      'danger-red': '#FC555619',
+      'un-assigned-bg': "#F7C2D2",
```

Comment:
rather than calling the colors, unassigned, assigned,etc instead name it by the color itself.
That way it makes sense when we re-use it.
ex. #F7C2D2 is light shade of pink. so name it pink-100; that way in future, I can easily re-use it by seeing the color codes.
else I won't know if pink shade already exists and I will end up creating another color shade of pink.

Idea is simple. The naming convention should clearly convey the meaning and help in making it reusable